webkit2/webkit2.web-context.lisp

```diff
@@ -148,7 +157,7 @@
   (hostname :string))
 (export 'webkit-web-context-prefetch-dns)
 
-(defcfun "webkit_web_context_set_disk_cache_directory" :void
+(defcfun "webkit_web_context_set_disk_cache_directory" :void ;; TODO: Deprecated since 2.10. Remove?
```

Comment:
As long as WebKitGTK keeps it in its library, we need to keep it as well, lest we break dependent code.
I suggest to add the deprecation warning to the cfun docstring.

Reply:
Hmmmm, this makes me worry about these deprecated `local-storage` things I removed in the other pull-request. They are still present in the C API, even though they're considered deprecated. I'll look into that!